src/api/db/models/utils.ts

```diff
@@ -134,11 +134,22 @@ export function buildPipeline(
     labels,
     reviewed,
     custom,
+    comments
   }: gql.Filters,
   projectId?: string,
 ): PipelineStage[] {
   const pipeline: PipelineStage[] = [];
 
+  if (comments) {
+    pipeline.push({ 
+      $match: {
+        $text: {
+          $search: `"${comments}"`
```

Comment:
I can't tell here if we need to wrap the comments in an additional set of escaped double quotes like in the [example](https://www.mongodb.com/docs/manual/reference/operator/query/text/#phrases) in the docs in order for it to search for the whole phrase or not.

Reply:
I believe when using backticks (`), it will treat the double quotes as double quotes and so we shouldn't need to escape them